package.json

```diff
@@ -4,8 +4,8 @@
   "scripts": {
     "start": ". ./scripts/populate-git-env.sh && next dev --port 8888",
     "start:tunnel": "ngrok http 8888",
-    "build": "yarn test:once:group:no-integration && next build",
-    "build:profiler": ". ./scripts/populate-git-env.sh && next build --profile --debug",
+    "build": ". ./scripts/populate-git-env.sh && yarn test:once:group:no-integration && next build",
```

Comment:
This might break CI/CD, why do you need to run `yarn build` locally?

Reply:
This will try to run the script in the Vercel environment, where git isn't defined, and it'll break.

Reply:
@samuelcastro Alternatively, we might add a `yarn build:local` that does call this env script, that'd probably be the easiest workaround, what do you think?

Reply:
This is not breaking CI, I've tested it. I like to build locally to improve my dev experience and also to gain time and avoid the need to push bad commits that otherwise would fail on CI.

Reply:
Yeah, build:local is way better.

Reply:
Okay! I'll make a quick PR for it then :)